front-end/components/Auth/LoginView.jsx

```diff
@@ -32,11 +36,25 @@ class LoginView extends React.Component {
           className='login-icon'
           src={loginIcon}
           alt="Login Icon" />
-        <h2>CTags</h2>
-        <img
-          src={rainbowButton}
-          alt="Login Button" />
-        <RoundedButton />
+        <h2 className='ctag-text'>CTags</h2>
+
+        <div style={{ height: '10vw' }}></div>
+
+        <div className='login-field'>
```

Comment:
I will just mention this again, but this div is redundant.  Also it is ambiguous because the child of this div has the same className so CSS will be inherited by both divs wether or not you want it to be.  If you did the following:

`
.login-field {
  margin: 10px
}
`

It would actually set the margin to 20px because it would apply to the child and the parent.  This can be challenging to debug.